content/news/2023-03-21-foodborne-training/index.md

```diff
@@ -0,0 +1,109 @@
+2. **compare different samples to track** the possible source of contamination
+The workflows are supported by a dedicated tutorial.
+
+## A tutorial for pathogen detection from (direct Nanopore) sequencing data using Galaxy
+
+Our tutorial material is available since January on the Galaxy Training Network: ["Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition"](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html)
+
+The training includes two versions:
+
+- A **Short Version**: where learners will run 5 prebuilt workflows, such that at the end they have the full workflow running.
+- A **Long Version**: Where a step by step, tool by tool, is presented along with its parameters for the learners to run
+
+The workflow or the step by step tools used in the training can be summaried as follows:
+
+![Pathogen Detection Workflow](./workflow.png)
+
+# Workshops
+Several workshops using this tutorial have been run or will be run in the next months.
+
+## A practical workshop for (foodborne) pathogen detection from (direct Nanopore) sequencing data using Galaxy
+
+A [__first workshop__](/events/2023-02-02-foodborne-pathogen-detection-workshop/) to test this training was conducted on the 2nd of February 2023 as a 1-day hybrid (F2F at FHNW Muttenz, Switzerland and online), free and practical workshop.
+
+**35** on-site and online participants from different background knowledge have attended the workshop. The workshop started with an introduction to Galaxy and the GTN, an introduction to the Foodborne pathogen detection project funded by EOSC-Life, and finally our new GTN material was explained.
+
+More than **90 people registered**, 53% male, with diverse career stages and locations:
+
+<img class="img-fluid float-left" src="./continents.png" style="width:400px;" alt="Participant continent: 10% Africa, 37% Europe, 8% North America, 36% Asia, 7% South America, 2% Oceania"/>
+
+<img class="img-fluid float-left" src="./career-stage.png" style="width:400px;" alt="Participants career stage: 7.9% Undergraduate, 1.1% Study director, 28.1% staff scientist, 2.2% researcher, 9.0% postdoctoral researchers, 27% PhD student, 3.4% Lab head, 16.9% Graduate student, 2.2% Assistant Professor"/>
+
+**63 confirmed** their registration and gave us more information about their motivation in participating in the workshop.
+
+To help prepare for the course, we also questioned them about their background in the generation of pathogen sequencing data, using Galaxy, but also about their confidence about the topics, here are their answers:
+
+<img class="img-fluid float-left" src="./motivation2.png" style="width:400px;" alt=""/>
```

Comment:
Could you make the alt text more functional? They are used by screen readers to explain the content of the images, so they should describe the image content. I did it for the 2 images above

Reply:
i will still add more in them, i just pushed now to save them